templates/vanilla/packages/client/tsconfig.json

```diff
@@ -11,10 +11,8 @@
     "isolatedModules": true,
     "esModuleInterop": true,
     "noEmit": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "noImplicitReturns": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "jsx": "react-jsx"
```

Comment:
with linked deps, this causes issues when importing dev-tools because it uses `.tsx` extensions (for things with JSX)